src/browser/index.jsx

```diff
@@ -219,30 +219,16 @@ var TabBar = React.createClass({
     var tabs = this.props.teams.map(function(team, index) {
       var unreadCount = 0;
       var badgeStyle = {
-        unread: {
-          background: '#383838',
-          float: 'right',
-          color: 'white',
-          textAlign: 'center',
-          marginTop: '5px',
-          width: '10px',
-          height: '10px',
-          marginLeft: '5px',
-          borderRadius: '50%',
-        },
-
-        mention: {
-          background: '#f1342c',
-          float: 'right',
-          color: 'white',
-          minWidth: '19px',
-          fontSize: '12px',
-          textAlign: 'center',
-          lineHeight: '20px',
-          height: '19px',
-          marginLeft: '5px',
-          borderRadius: '50%',
-        }
+        background: '#FF1744',
```

Comment:
Yep, changed to match https://github.com/mattermost/desktop/blob/master/src/browser/js/badge.js#L12 which we show as an task bar overlay